utils/fetch_sdl2_framework.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 # Download and extract the SDL v2 framework for macOS.
 
-SDL2_VERSION="2.0.14"
+SDL2_VERSION="2.0.22"
```

Comment:
I think it's fine to keep this change separate. (I'm planning to cherry-pick this PR back to a 0.9.14 hotfix for the long-awaited steam "stable release" of 0.9.14 in the next two weeks.)

Reply:
I did notice that with current SDL2, the game window no longer flashes red just before showing the initial screen on older macOS versions.

Reply:
I've updated the rest of this script to ensure a fresh copy of the framework is downloaded & copied if the build number is changed. If not, and a framework is already in `build`, the script phase is skipped entirely.